downstream/assemblies/platform/assembly-install-aap-operator.adoc

```diff
@@ -11,6 +11,10 @@ ifdef::context[:parent-context: {context}]
 
 .Prerequisites
 * You have installed the {PlatformName} catalog in Operator Hub.
+* You have created a Storage Class if one does not exist. See link:https://docs.openshift.com/container-platform/4.10/storage/dynamic-provisioning.html[Dyamic Provisioning] for details.
+* To run {OCP} clusters on Amazon Web Services with ReadWriteMany, you must add NFS or other storage. 
+** For information on the AWS Elastic Block Store (EBS) or to use the `aws-ebs` storage class, see link:https://docs.aws.amazon.com/eks/latest/userguide/storage-classes.html[Storage Classes].
+** To use multi-attach ReadWriteMany for AWS EBS, see link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-volumes-multi.html[Attaching a volume to multiple instances with Amazon EBS Multi-Attach].
```

Comment:
Do we really want to point to 3rd party docs? I don't think that is recommended.

OCP docs have lots of platform specific details, including AWS: https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html-single/storage/index#persistent-storage-aws